pymobiledevice3/services/crash_reports.py

```diff
@@ -12,6 +13,7 @@
 class CrashReportsManager:
     COPY_MOBILE_NAME = 'com.apple.crashreportcopymobile'
     CRASH_MOVER_NAME = 'com.apple.crashreportmover'
+    DEFAULT_FOLDER = '/com.apple.appstored'
```

Comment:
the name "default" doesn't really explain this. just call it `APPSTORED_PATH` and just before referring it explain why this is a special case

Reply:
done